ocp_cleanup.sh

```diff
@@ -4,8 +4,10 @@ set -e
 
 source ocp_install_env.sh
 
-$GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug --dir ocp destroy cluster || true
-
-rm -rf ocp
-
-sudo rm -rf /etc/NetworkManager/dnsmasq.d/openshift.conf
+if $GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug --dir ocp destroy cluster; then
+  rm -rf ocp
+  sudo rm -rf /etc/NetworkManager/dnsmasq.d/openshift.conf
+else
+  echo "Error running openshift-install destroy cluster, not cleaning ./ocp or /etc/NetworkManager/dnsmasq.d/openshift.conf"
```

Comment:
```suggestion
 >&2 echo "Error running openshift-install destroy cluster, not cleaning ./ocp or /etc/NetworkManager/dnsmasq.d/openshift.conf"
```
Better print errors to stderr